packages/yoastseo/src/scoring/assessments/SCORING READABILITY.md

```diff
@@ -32,13 +53,13 @@
 
 **Call to action URL**: [https://yoa.st/35e](https://yoast.com/paragraph-length-check/#utm_source=yoast-seo&utm_medium=software&utm_term=paragraph-length-cta&utm_content=content-analysis) (link placement is in bold in the feedback strings)
 
-| Bullet 	 | Score	| Criterion | Feedback |
+| Traffic light 	 | Score	| Criterion | Feedback |
 |------------|------------------	|--------------------- |---------------------
 | Red        |3	                |> 200 words	   	          |**Paragraph length**: X of the paragraphs contain more than the recommended maximum of X words. **Shorten your paragraphs!**|
 | Orange     |6                 |Between 150 and 200 words    |**Paragraph length**: X of the paragraphs contain more than the recommended maximum of X words. **Shorten your paragraphs!**|
 | Green      |9                 |≤ 150 words	              |**Paragraph length**: none of the paragraphs are too long. Great job! |
 
-## 3) Sentence length
+### 3) Sentence length
 **What it does**: Checks whether the sentences exceed the recommended maximum length (default: 20 words, IT: 25 words, RU: 15 words, HE: 15 words, TR: 15 words)
 
 **When applies**: When the text (cleaned with sanitizeText) has more than 50 characters. This is tied to the TextPresenceAssessment.
```

Comment:
I wonder if this information is actually useful for the users? what do we mean by `sanitizeText`. Though I am also fine as is, since it's not in the scope of your issue to improve this